_data/prefixes.yaml

```diff
@@ -9,3 +9,4 @@ qudt: http://qudt.org/schema/qudt#
 ignf: http://data.ign.fr/def/ignf#
 inspiregloss: http://inspire.ec.europa.eu/glossary/
 rel: http://www.iana.org/assignments/relation/
+metocean: https://api.meteogate.eu/profiles/metocean/covjson
```

Comment:
Just checking we think this is sensible. "metocean" is a term in widespread use, and it makes sense if the URI is somehow authoritative within that community as a whole. If the URI is specific to a certain project or narrower context, then may be a different prefix (e.g. "meteogate") might be better?